Library/Formula/openvpn.rb

```diff
@@ -11,11 +9,46 @@ class Openvpn < Formula
     sha1 "bddff060d3d3e70db96abcf6ccc95952cdb7a99c" => :mountain_lion
   end
 
+  option "with-easy-rsa", "Build with easy-rsa"
+
+  if build.with? "easy-rsa"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   depends_on "lzo"
   depends_on :tuntap
   depends_on "openssl"
 
+  resource "easy_rsa" do
+    url "https://github.com/OpenVPN/easy-rsa/archive/2.2.2.tar.gz"
+    sha256 "38f1f84e9b966df8b312365c7c75a455bdf003abdd5156e7106fa9c7771583ae"
+  end
+
   def install
+    if build.with? "easy-rsa"
+      resource("easy_rsa").stage do
+        system "autoreconf", "-fvi"
+        system "./configure", "--prefix=#{libexec}", "--disable-silent-rules"
+        system "make", "install"
+
+        bin.mkpath
+        File.rename libexec/"share/easy-rsa/vars", libexec/"share/easy-rsa/vars.sample"
+        bins = Dir["#{libexec}/share/easy-rsa/*"].select { |f| File.executable? f }
+
+        Dir.glob(bins) do |f|
+          execname = File.basename(f, "")
+          (bin+execname).write <<-EOS.undent
+            #!/usr/bin/env bash
+            export EASY_RSA=#{libexec}/share/easy-rsa
+            #{f} "$@"
+          EOS
+          chmod 0755, bin+execname
```

Comment:
Yeah, that should work, thanks. Will that take care of the ` chmod ` as well or?